indico/modules/events/editing/schemas.py

```diff
@@ -162,3 +162,20 @@ def _check_for_unique_tag_code(self, code):
             query = query.filter(EditingTag.id != tag.id)
         if query.has_rows():
             raise ValidationError(_('Tag code must be unique'))
+
+class EditableFileTypeArgs(mm.Schema):
+    class Meta:
+        rh_context = ('event', 'file_type')
+
+    name = fields.String(required=True, validate=not_empty)
+    extension = fields.String(required=True, validate=not_empty)
```

Comment:
that's a list of extensions -> `fields.List(...)`

and it may be empty, so no need for any validate